charts/backoffice/templates/tests/stern-integration.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-stern-integration"
+  annotations:
+    "helm.sh/hook": test
+  labels:
+    app: stern-integration
+    release: {{ .Release.Name }}
+spec:
+  volumes:
+    - name: "stern-integration"
+      configMap:
+        name: "stern-integration"
+    # Used to read some values from the stern service
+    - name: "stern-config"
+      configMap:
+        name: "stern"
+  containers:
+  - name: integration
+    image: "{{ .Values.image.repository }}-integration:{{ .Values.image.tag }}"
+    volumeMounts:
+    - name: "stern-integration"
+      mountPath: "/etc/wire/integration"
+    - name: "stern-config"
+      mountPath: "/etc/wire/stern/conf"
+    resources:
+      requests:
+        memory: "512Mi"
+        cpu: "2"
```

I think this doesn't need as much resources. How about:

```suggestion
        memory: "128Mi"
        cpu: "1"
```